### DISCUSSION TITLE
Github stats broken after changing my profile name

### DISCUSSION THREAD

**Discussion by Pareek-Yash** [community/community#42376] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,
It seems GitHub stats is broken after changing my profile name. This is not my profile name now.
Seems the new profile name is not updated in stats dept
![somethingwentwrong](https://user-images.githubusercontent.com/59692875/209138675-a6849d45-acf4-417b-8c9b-d08b4733647d.PNG)

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @Pareek-Yash 

Your username is wrong in the README file, replace "deepAIYash" with "Pareek-Yash "

![image](https://user-images.githubusercontent.com/47532703/209167730-3c9236e0-6f0a-4ee3-9633-f7ed487d1d25.png)

**Reply by Pareek-Yash** [reply]
Thank you.I think I should look for a fix next time instead of just asking questions next time.

**Reply by SilasRodrigues19** [reply]
No problem.

If it worked, please mark as answered.
